package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "chronofactorem",
+  "version": "0.1.0",
+  "type": "module",
+  "description": "Timetable maker-helper",
+  "homepage": "https://github.com/Dryft-bits/ChronoFactorem",
+  "author": "Dryft-bits",
+  "main": "server.js",
+  "dependencies": {
+    "body-parser": "^1.18.3",
+    "config": "^3.2.5",
+    "debug": "^4.1.0",
+    "dotenv": "6.2.0",
+    "express": "^4.17.1",
+    "express-validator": "^6.4.0",
+    "mongoose": "^5.8.11",
+    "node-sass": "^4.5.3",
+    "node-sass-middleware": "^0.11.0",
+    "react": "^16.6.3",
+    "react-dom": "^16.6.3"
+  },
```

Comment:
`bcryptjs` and `jsonwebtoken` were for implementing staff auth functions (encryption and stuff). 
Have any alternative in mind?

Reply:
The login module will be provided by the TA na? Let's see if that one won't have the stuff for staff login included

Reply:
Iirc, Sir said the **google** login implementation would be provided by the TA. Anyways, the rest looks good.

Reply:
I mean the google login would also probably use some other library. So we can just reuse that one. Or we can add these later if it doesn't work out